app/controllers/border/BorderModeOfTransportController.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.border
```

Comment:
Can we update the directory to be transport.transportMeans?

Reply:
Updated the test location too

Reply:
Hello Joe, this seems to have been left in the `border` directory, we can move this to controllers.transport.transportMeans.